scripts/build.sh

```diff
@@ -3,21 +3,26 @@
 set -xe
 
 echo "***** Start: Building Miniforge installer *****"
-
 CONSTRUCT_ROOT="${CONSTRUCT_ROOT:-$PWD}"
 
 cd $CONSTRUCT_ROOT
 
 # Constructor should be latest for non-native building
 # See https://github.com/conda/constructor
 echo "***** Install constructor *****"
-conda install -y "constructor>=3.1.0" jinja2 -c conda-forge -c defaults --override-channels
+if [[ "${TARGET_PLATFORM}" == "linux-ppc64le" ]]; then
+  USE_DEFAULTS="-c defaults"
+fi
+
+conda install -y "constructor>=3.1.0" jinja2 curl libarchive -c conda-forge $USE_DEFAULTS --override-channels
+
+
 if [[ "$(uname)" == "Darwin" ]]; then
-    conda install -y coreutils -c conda-forge -c defaults --override-channels
+    conda install -y coreutils -c conda-forge --override-channels
 elif [[ "$(uname)" == MINGW* ]]; then
-    conda install -y "nsis=3.01" -c conda-forge -c defaults --override-channels
+    conda install -y "nsis=3.01" -c conda-forge --override-channels
 fi
-pip install git+git://github.com/conda/constructor@3.2.1#egg=constructor --force --no-deps
+pip install git+git://github.com/wolfv/constructor@69eaa20b12ea1020438f645ec834f47ac9d3c3a7#egg=constructor --force --no-deps
```

Comment:
Asked upstream about getting this change reviewed ( https://github.com/conda/constructor/pull/450 )